aws-sso-instanceaccesscontrolattributeconfiguration/docs/README.md

```diff
@@ -43,18 +46,26 @@ _Maximum_: <code>1224</code>
 
 _Pattern_: <code>arn:aws:sso:::instance/(sso)?ins-[a-zA-Z0-9-.]{16}</code>
 
-_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+_Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)
 
 #### InstanceAccessControlAttributeConfiguration
 
 InstanceAccessControlAttributeConfiguration for  sso instance
 
-_Required_: Yes
```

Comment:
Shouldn't this be "Required: Conditional", specifying that either one or the other needs to be set? This is how other resource docs do it, e.g. [`AWS::CloudFormation::Authentication`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-authentication.html)

Reply:
It's also probably worth explaining why the change was made, and that (I hope) future properties will only be added at the top level, rather than in both places.

Reply:
My bad those files is autogenerated. Every time I am building this project default files will be regenerated.
I'll update this pr.

Reply:
@benkehoe
> It's also probably worth explaining why the change was made, and that (I hope) future properties will only be added at the top level, rather than in both places.

+1 - we do have a question though.  What is the customer preference with CFN changes which aim to deprecate a CFN resource?  We can:

1. Remove `InstanceAccessControlAttributeConfiguration` completely, which is a bit of a fib but sends the right message to use `AccessControlAttributes` instead.
2. Keep `InstanceAccessControlAttributeConfiguration` and `AccessControlAttributes` and do what they did in [AWS::CloudFormation::Authentication](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-authentication.html) by having them set to _Required_: Conditional. Which is also a bit of a fib because `InstanceAccessControlAttributeConfiguration` is deprecated per the [resource documentation](https://github.com/andriiglukhyi/aws-cloudformation-resource-providers-sso/blob/master/aws-sso-instanceaccesscontrolattributeconfiguration/docs/instanceaccesscontrolattributeconfiguration.md)
3. Have `InstanceAccessControlAttributeConfiguration` in the top level readme.md but mention it's deprecation.  This would allow us to provide that explanation you're looking for and set the tone for the next updates being made at the top level instead of continued nesting

Thoughts?

Reply:
I think a combination of 1 and 3 would be something like what you have currently in the PR (`InstanceAccessControlAttributeConfiguration` is not in the schema), plus "The top-level attribute `InstanceAccessControlAttributeConfiguration` has been deprecated, but its schema can be viewed here" as a note that links to the schema page for that attribute.

I think my answer would probably be option 2 if this had been out in the wild for a while, where `AccessControlAttributes` might say "Required: Yes (unless deprecated field `InstanceAccessControlAttributeConfiguration` is in use, see deprecation notice)".

Reply:
To clarify since this might be confusing, [official CloudFormation PUBLIC type reference documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-template-resource-type-ref.html) is not pulled from these `docs/` autogenerated by [`cloudformation-cli`](https://github.com/aws-cloudformation/cloudformation-cli) if that's the main concern

@iann0036 [added autogenerated `docs/` here](https://github.com/aws-cloudformation/cloudformation-cli/pull/367), which is especially useful for non-PUBLIC CloudFormation Registry types. [Official CloudFormation PUBLIC type reference documentation is hosted in this repository](https://github.com/awsdocs/aws-cloudformation-user-guide/)